stable/drupal/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: drupal
-version: 0.4.6
+version: 0.4.7
```

Comment:
That's probably what @seanknox meant. A minor bump in a dependency doesn't necessarily mean we need a minor bump here. Question is, how can we know best when to do what?